charts/controller/templates/controller-deployment.yaml

```diff
@@ -128,6 +128,10 @@ spec:
                 secretKeyRef:
                   name: database-creds
                   key: password
+{{- if (.Values.deis_ignore_scheduling_failure) }}
+            - name:DEIS_IGNORE_SCHEDULING_FAILURE
```

Comment:
Can you add a space here?

Reply:
If we are adding this to `.Values`, we should also add it to this file also to set a proper default: https://github.com/teamhephy/controller/blob/master/charts/controller/values.yaml

Reply:
A few of these env vars seem to use "if they exist, then write them" so they don't need a default, and wont appear in the file unless they are added. That's the method I used.

Reply:
That works too! Was not sure if we should be more explicit but since we added this var to the documentation for the controller. Maybe we don't need to be explicitly setting it in the Values file. :+1:

Reply:
I also copied the formatting from the {{- if (.Values.deploy_hook_urls) }} section.
I'm not sure where you would like a space...

Reply:
Right after `name:`, seems to be missing a space right after the colon and before the variable name, not sure if it matters.

Reply:
Ah Gotcha. Yes - that matters :) Fixing, thanks